modules/templates/helper.go

```diff
@@ -295,7 +295,7 @@ func ActionIcon(opType int) string {
 	switch opType {
 	case 1, 8: // Create and transfer repository
 		return "repo"
-	case 5, 9: // Commit repository
+	case 5, 9, 16, 17: // Commit repository
```

Comment:
Could we please use the named constants (e.g. `ActionDeleteTag`) instead of magic numbers here?

Reply:
Of course.
Should it be done in this PR or should it be a separate one because it is not really part of this one?

Reply:
I think it should be done in a new PR as it requires more changes in other parts of the code